src/main/java/mate/academy/hibernate/relations/dao/impl/ActorDaoImpl.java

```diff
@@ -12,11 +16,38 @@ public ActorDaoImpl(SessionFactory sessionFactory) {
 
     @Override
     public Actor add(Actor actor) {
-        return null;
+        SessionFactory sessionFactory = HibernateUtil.getSessionFactory();
```

Comment:
It is unnecessary to retrieve the `SessionFactory` from `HibernateUtil` again here, as it is already being passed to the constructor and can be accessed via `super.sessionFactory`. Consider using the `sessionFactory` field from the superclass instead.